ICAFIX/hcp_fix_multi_run

```diff
@@ -379,116 +387,365 @@ log_Msg "PWD: $DIR"
 # AND since 'addpath' *prepends* to the matlab path, the version in HCPCIFTIRWDIR will therefore take precedence.
 ML_PATHS="addpath('${HCPPIPEDIR}/global/fsl/etc/matlab'); addpath('${HCPCIFTIRWDIR}'); addpath('${HCPPIPEDIR}/global/matlab/icaDim'); addpath('${HCPPIPEDIR}/global/matlab'); addpath('${this_script_dir}/scripts');"
 
-## ---------------------------------------------------------------------------
-## Preparation (highpass) on the individual runs
-## ---------------------------------------------------------------------------
+## ====================================================================================
+## RunICA unless RunFIXOnly is set to TRUE. In the latter case skip all steps until FIX 
+## ====================================================================================
 
-#Loops over the runs and do highpass on each of them
-log_Msg "Looping over files and doing highpass to each of them"
-
-NIFTIvolMergeARRAY=()
-NIFTIvolhpVNMergeARRAY=()
-SBRefVolARRAY=()
-MeanVolARRAY=()
-VNVolARRAY=()
-CIFTIMergeARRAY=()
-CIFTIhpVNMergeARRAY=()
-MeanCIFTIARRAY=()
-VNCIFTIARRAY=()
-MovementNIFTIMergeARRAY=()
-MovementNIFTIhpMergeARRAY=()
-MovementTXTMergeARRAY=()
+if ((!SkipICA))
```

Comment:
At least some versions of bash don't like this, because `!` is also a history special character.  Add a space after the `!`.
```suggestion
if ((! SkipICA))
```

Reply:
Done.